### PR TITLE
Add ntia-conformance-checker compliance selector to UI

### PR DESCRIPTION
**Pull request by bact**
Changes to ntia-conformance-checker related code, to prepare it for [more compliance checkers](https://github.com/spdx/ntia-conformance-checker/pull/224) in the future.

These changes are effectively only in the UI side for now and there's nothing change on the backend side apart from the upgrade of the library.

- Update ntia-conformance-checker version to 3.0.2
- Add compliance selector to UI
  - There's only one option now, which is the selected default "ntia"
- Add a (commented) code to get compliance request value
  - If the value is not exist, use "ntia" as default
  - This value will be used post-3.0.2 to let SbomChecker knows which compliance it should check against
- Remove unused imports
- Fix few typos

### CODE REVIEW

**Review by goneall — APPROVED** [review]
LGTM

File: requirements.txt
Comment (goneall) [inline requirements.txt:18]:
```suggestion
spdx-tools==0.8.3
```

NTIA tools version 3.0.2 requires an update to the spdx-tools to 0.8.3

**Review by goneall — CHANGES_REQUESTED** [review]
I'll do some more testing - may add further comments, but there is one change I found needs to be made with the requirements.txt file.

File: src/app/templates/app/ntia_conformance_checker.html
Comment (goneall) [inline src/app/templates/app/ntia_conformance_checker.html:21]:
```suggestion
```

This option isn't supported by the Python libraries - we should remove it from the dropdown options.

File: src/app/templates/app/ntia_conformance_checker.html
Comment (goneall) [inline src/app/templates/app/ntia_conformance_checker.html:22]:
```suggestion
```

This option isn't supported by the Python libraries - we should remove it from the dropdown options.

File: src/app/templates/app/ntia_conformance_checker.html
Comment (goneall) [inline src/app/templates/app/ntia_conformance_checker.html:26]:
```suggestion
```

This option isn't supported by the Python libraries - we should remove it from the dropdown options.

**Review by goneall — CHANGES_REQUESTED** [review]
A few more changes to fix problems that were present in the previous versions - might as well fix them with this update.

**Review by goneall — COMMENTED** [review]
LGTM - It looks like we're about to get a new release of the conformance checker.

@bact - do you want to wait for the new release before merging?

**Review by goneall — APPROVED** [review]
Everything tests out fine, so I'm going to go ahead and merge this PR.

If we decide to include a new release of the NTIA Conformance Checker, we can create a second PR.

**Comment by bact** [comment]
Thank you. I will create another PR after we have the new release of the checker.

**Pull request merged**
